cmd/loadtest/loadtest.go

```diff
@@ -963,9 +970,32 @@ func loadtestTransaction(ctx context.Context, c *ethclient.Client, nonce uint64)
 	chainID := new(big.Int).SetUint64(*ltp.ChainID)
 	privateKey := ltp.ECDSAPrivateKey
 
+	head, err := c.HeaderByNumber(ctx, nil)
+	if err != nil {
+		log.Error().Err(err).Msg("Unable to get head")
+		return
+	}
```

Comment:
This seems like it could have a lot of overhead. Eg if we're performing a load test with 2500 transactions per second and 2 second block time, you would fetch the header 5000 times, but it would have only changed once.